mlflow/tracing/display/display_handler.py

```diff
@@ -21,6 +71,27 @@ def _serialize_trace_list(traces: list["Trace"]):
     )
 
 
+def _get_query_string_for_traces(traces: list["Trace"]):
+    query_params = []
+
+    for trace in traces:
+        query_params.append(("trace_id", trace.info.request_id))
+        query_params.append(("experiment_id", trace.info.experiment_id))
+
+    return urlencode(query_params)
+
+
+def is_using_tracking_server():
+    return is_http_uri(mlflow.get_tracking_uri())
+
+
+def validate_environment():
+    # the notebook display feature only works in
+    # Databricks notebooks, or in Jupyter notebooks
+    # with a tracking server
+    return is_in_databricks_runtime() or is_using_tracking_server()
```

Comment:
Do we need to add `is_jupyter()` here?

Reply:
yeah actually let me add a helper function for that